blocks/video-player/video-player.js

```diff
@@ -0,0 +1,36 @@
+function loadVideo(videoURL, block) {
+  const VIDEO_HOST = 'https://www.youtube.com';
```

Comment:
This limits supported video URLs to youtube. should we be clear and rename the `video-player` to `youtube-player`? Also, since the constant is only used once, i'd be ok with putting the value directly in line 16 for readability.

Reply:
embedding works differently for each video provider, maybe we could ask if they only want to use youtube or if they plan on using other providers like vimeo etc. Then we could have a generic video-player, that just composes the embed url from the iframe differently

Reply:
i asked the customer on slack